mythtv/programs/mythfrontend/manualschedule.cpp

```diff
@@ -75,6 +77,9 @@ bool ManualSchedule::Create(void)
         m_chanids.push_back(channels[i].m_chanId);
     }
 
+    if (last_manual_chan != -1)
+        m_channelList->SetItemCurrent(last_manual_chan);
+
```

Comment:
https://www.mythtv.org/wiki/Coding_Standards#Spaces_and_braces " One-line `if` or `else` bodies should have braces."